src/components/Search.js

```diff
@@ -12,19 +14,25 @@ class Search extends Component {
     }
 
     handleCountySelect = (event) => {
-        axios.get(event.path)
-            .catch(error => {
-                console.log('error', error);
-            })
-            .then(response => {
-                // console.log('event', event)
-                // console.log('response', response)
-                const json = response.data;
-                this.props.onCountySelect(json);
-                this.setState({
-                    'countyName': event.name + ', ' + event.usps
-                })
-            });
+        this.props.onCountySelect(event);
+        this.setState({
+            'countyName': event.name + ', ' + event.usps
```

Comment:
I believe you can do string formatting like this
`${event.name}, ${event.usps}`

Reply:
`'countyName'` should be countyName. They both work but the editor will be able to handle it if you use the key here